compiler/rustc_index/src/vec.rs

```diff
@@ -116,7 +130,7 @@ impl<I: Idx, T> IndexVec<I, T> {
     }
 
     #[inline]
-    pub fn from_elem<S>(elem: T, universe: &IndexVec<I, S>) -> Self
+    pub fn from_elem<S>(elem: T, universe: &IndexSlice<I, S>) -> Self
```

Comment:
It would be nice to have documentation here, I don't really get why/how you'd use this method...

Reply:
Send as PR #109913